### PR TITLE
ATC aug 2024 refresh

### PR DESCRIPTION
**Pull request by ratuat**

### CODE REVIEW

File: ATC/manual work/readme.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    concept_name,
+    form,
+    atc_code,
+    atc_name,
+    rx_ids,
+    rx_names
+FROM norske_result;
+```
+* extract the [respective tsv file](https://drive.google.com/file/d/1_QOGi9foXuo6EWX3HWKDl_fwfSdQFzAF/view?usp=drive_link) into the *zindex_full* table. The file was generated using the query:
+```sql
```
Comment (TinyRickC137) [inline ATC/manual work/readme.md:57]: Should not be available in the final assembly - proprietary data

File: ATC/load_stage.sql [hunk 2]
```diff
@@ -1,1173 +1,375 @@
...
+        FROM new_atc_codes_ings_for_manual
+        WHERE relationship_id in ('ATC - RxNorm pr lat', 'ATC - RxNorm sec lat')
+      ) t1
+      JOIN devv5.concept t2
+          ON t1.concept_code_2::int = t2.concept_id and t2.vocabulary_id in ('RxNorm', 'RxNorm Extension');
+
+
+--6. Insert ATC - RxNorm relationships
+DROP TABLE IF EXISTS  new_unique_atc_codes_rxnorm;
+CREATE TABLE new_unique_atc_codes_rxnorm AS
```
Comment (AlaikseiKatyshou) [inline ATC/load_stage.sql:266]: Suggest creating temporary or unlogged table, like: 
CREATE UNLOGGED TABLE
or
CREATE TEMP UNLOGGED TABLE

File: ATC/load_stage.sql
Comment (AlaikseiKatyshou) [inline ATC/load_stage.sql:1]: Suggest bringing the code to the [codestyle](https://ohdsi.github.io/Hades/codeStyle.html#OHDSI_code_style_for_SQL)

**Review by AlaikseiKatyshou — COMMENTED** [review]
reviewed

File: ATC/Archive/load_stage.sql
[the remark this answers is not in the document]
Comment (ratuat) [reply ATC/Archive/load_stage.sql:1]: For sure not, just temporary decision.

File: ATC/load_stage.sql (hunk 2 above)
Comment (ratuat) [reply ATC/load_stage.sql]: Corrected.

File: ATC/manual work/readme.md (hunk 1 above)
Comment (ratuat) [reply ATC/manual work/readme.md]: Done

File: ATC/Archive/load_stage.sql
[the remark this answers is not in the document]
Comment (TinyRickC137) [reply ATC/Archive/load_stage.sql:1]: We need to keep them somehow. Maybe create a protected branch? And we will document all the related branches (current master and 2021 refresh)

**Review by AlaikseiKatyshou — APPROVED** [review]

**Pull request merged**